app/[locale]/(dashboard)/checkout/page.tsx

```diff
@@ -0,0 +1,59 @@
+import { getCart } from "../../../api/api";
+import Image from "next/image";
+import DeleteAll from "../../components/checkOutButtons/DeleteAll";
+import IncrementProductIcon from "../../components/checkOutButtons/IncrementProductIcon";
+import DecrementProductIcon from "../../components/checkOutButtons/DecrementProductIcon";
+import DeleteIcon from "../../components/checkOutButtons/DeleteIcon";
+export const revalidate = 0;
+
+export default async function CheckOut() {
+  const cart = await getCart("29");
+  const cartItems = await cart.json();
+  //   console.log(cartItems, "items");
+
+  const productQuantityMap: ProductQuantityMap = [];
+  cartItems.rows.forEach((item: CartItem) => {
+    productQuantityMap[item.productid] = item.quantity;
+  });
+
+  const productPromises = cartItems.rows.map((item: Product) =>
```

Comment:
item should be type of CartItem and the build problem would be resolved

Reply:
okey got it

Reply:
can u check it now?